src/engine/BMGame.php

```diff
@@ -4084,7 +3851,7 @@ protected function setBMPlayerProps($property, array $value, $subclass = '') {
     /**
      * Set all players' waitingOnAction statuses to FALSE
      */
-    protected function setAllToNotWaiting() {
+    public function setAllToNotWaiting() {
```

Comment:
Okay, let's make that another issue then: that BMInterface ideally shouldn't directly call $game->log_action().